.github/workflows/pull-request.yaml

```diff
@@ -73,6 +73,7 @@ jobs:
 
         replicated release create --yaml-dir e2e/kots-release-install --promote CI --version "${SHORT_SHA}"
         replicated release create --yaml-dir e2e/kots-release-upgrade --promote CI --version "${SHORT_SHA}-upgrade"
+        replicated release create --yaml-dir e2e/kots-release-install-old-version --promote CI --version "${SHORT_SHA}-old-version"
```

Comment:
why not create the 'old release' once with a version like 'pre-minio-removal', and then download that version from the endpoint?